src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public enum Colors {
+    black,
+    white,
+    red,
+    blue
```

Comment:
Enum is a set of constant values so you should follow the naming rules for constant (CONSTANT_CASE: all uppercase letters, with each word separated from the next by a single underscore).